tools/utils/tokenizer.lua

```diff
@@ -58,12 +62,12 @@ local options = {
       is learnt using `learn_bpe.lua`.]]
   },
   {
-    '-EOT_marker', separators.EOT,
-    [[Marker used to mark the end of token.]]
+    '-bpe_EOT_marker', separators.EOT,
+    [[Marker used to mark the End of Token while applying BPE in mode 'prefix' or 'both'.]]
```

Comment:
why changing the name of this option?

Reply:
To make it clear as a bpe related option and then for regex based filtering to not include these bpe options in learn_bpe.lua while tokenizing

Reply:
ok - put a note in *BREAKING CHANGES* section although I don't think it had ever been used. Please prepare also for documentation update and see with Guillaume to merge the PR.